mysql-test/suite/parts/t/optimizer.test

```diff
@@ -47,3 +47,14 @@ SHOW status LIKE 'handler_read_key';
 --enable_ps2_protocol
 
 DROP TABLE t1, t2;
+
+--echo #
+--echo # Bug#28509: Access null pointer during add_key_field call
```

Comment:
Please use  `MDEV-28509`.  That's what people grep for.   `Bug#nnnnn` notation was used for bugs at MySQL Ab, people may think you're refererring to https://bugs.mysql.com/bug.php?id=28509